modules/epang/place/main.nf

```diff
@@ -0,0 +1,41 @@
+process EPANG_PLACE {
+    tag "$meta.id"
+    label 'process_high'
+
+    conda (params.enable_conda ? "bioconda::epa-ng=0.3.8" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/epa-ng:0.3.8--h9a82719_1':
+        'quay.io/biocontainers/epa-ng:0.3.8--h9a82719_1' }"
+
+    input:
+    tuple val(meta), path(queryaln), path(referencealn), path(referencetree)
```

Comment:
Do I understand correctely that `queryaln` is essentially the sequences to be placed (i.e. samples), and `referencealn` & `referencetree` reference files?
If yes, I'd assume that `referencealn` & `referencetree` are sample independent and therefore I would suggest
```suggestion
    tuple val(meta), path(queryaln)
    path(referencealn)
    path(referencetree)
```

Reply:
Your half correct. In most cases I think the reference alignment and tree will differ between samples. But I changed anyway.